testing/eval_utils.py

```diff
@@ -21,15 +21,17 @@ def calculate_adtm(results: Dict) -> Dict:
         count = len(metrics)
         metric_best = max(metrics)
         metric_worst = min(metrics)
-        for metric in metrics:
-            adtm_sum += (metric - metric_worst) / (metric_best - metric_worst)
-        adtm_scores[model_name] = adtm_sum / count if count > 0 else 0
+        if metric_best == metric_worst:
+            adtm_scores[model_name] = 0
+        else:
+            for metric in metrics:
+                adtm_sum += (metric - metric_worst) / (metric_best - metric_worst)
```

Comment:
shouldn't it be metric - metric_best instead of worst. ADTM measures distance to the best metric